pheweb/serve/templates/layout.html

```diff
@@ -75,10 +75,19 @@
         <!-- Collect the nav links, forms, and other content for toggling -->
         <div class="collapse navbar-collapse" id="navbar-collapse">
             <ul class="nav navbar-nav navbar-right">
-                <li><a href="/random" style="color: #333; font-weight: bold;">Random</a></li>
-                <li><a href="/top_hits" style="color: #333; font-weight: bold;">Top Hits</a></li>
-                <li><a href="/coding" style="color: #333; font-weight: bold;">Coding Variants</a></li>
-                <li><a href="/static/FINNGEN_ENDPOINTS_DF4_V2.zip" style="color: #333; font-weight: bold;">Download Endpoint Definitions</a></li>
+              <li><a href="/random" style="color: #333; font-weight: bold;">Random</a></li>
+		{% if config.lof %}
```

Comment:
this is great.... potential to show something when it exists and leave conf out and it dissappears!